known_servers/known_servers-development-v1.2.0.json

```diff
@@ -0,0 +1,12 @@
+{
+    "active_servers": [
+        "transport.transport05.raiden.network"
+    ],
+    "all_servers": [
+        "transport.transport01.raiden.network",
+        "transport.transport02.raiden.network",
+        "transport.transport03.raiden.network",
+        "transport.transport04.raiden.network",
+        "transport.transport05.raiden.network"
+    ]
+}
```

Comment:
Basically it's the outcome of https://github.com/raiden-network/raiden/issues/6443.
We need a separate list for the Raiden clients and the server to server federation whitelist. 
Incidentally this also fixes the problem of onboarding new RSBs since we can add them to the `all_servers` list (so they become whitelisted for the federation) without making them "visible" to Clients yet until they have fully joined the federation.